src/components/organizations/editOrganizationFields.ts

```diff
@@ -0,0 +1,12 @@
+import { mapProperties } from "../../utils/list";
+import { generalOrgFields } from "./generalOrgFields";
+import { nestedFieldInArray } from "../forms/nestedFieldInArray";
+
+export const editOrganizationFields = 
+    matchingHref => 
```

Comment:
here we define the fields that live atop the plain json "state", providing ability to read and update the underlying json from an input text or other field; where updates cause a re-render with a new state.

in this example, we see `generalOrgFields` are imported. these are the "generic" definitions of fields for an org - title, homepage, mission... but we know that we have two uses for these "generic" fields - one is for the "new org" case, so those generic fields are applied to `state.possiblNewOrganization` object; whereas in this case - the "edit" case, those generic fields are applied to `state.organizations`, which is an array of orgs - here we indicate that we want these fields to apply to the record in the organizations array where the href matches the current href.